NPE

```
java.lang.RuntimeException: Unable to start service com.emmanuelmess.clock2.timers.TimerNotificationService@8875ddf with Intent { cmp=com.emmanuelmess.clock2.debug\/com.emmanuelmess.clock2.timers.TimerNotificationService (has extras) }: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.emmanuelmess.clock2.chronometer.ChronometerNotificationThread.updateNotification(boolean)' on a null object reference\n\tat android.app.ActivityThread.handleServiceArgs(ActivityThread.java:3045)
at android.app.ActivityThread.-wrap17(ActivityThread.java)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1452)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:148)
at android.app.ActivityThread.main(ActivityThread.java:5443)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:728)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:618)\nCaused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.emmanuelmess.clock2.chronometer.ChronometerNotificationThread.updateNotification(boolean)' on a null object reference
at com.emmanuelmess.clock2.chronometer.ChronometerNotificationService.updateNotification(ChronometerNotificationService.java:356)
at com.emmanuelmess.clock2.timers.TimerNotificationService.handleDefaultAction(TimerNotificationService.java:201)
at com.emmanuelmess.clock2.chronometer.ChronometerNotificationService.onStartCommand(ChronometerNotificationService.java:218)
at com.emmanuelmess.clock2.timers.TimerNotificationService.onStartCommand(TimerNotificationService.java:171)
at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:3028)
... 8 more\njava.lang.NullPointerException: Attempt to invoke virtual method 'void com.emmanuelmess.clock2.chronometer.ChronometerNotificationThread.updateNotification(boolean)' on a null object reference
at com.emmanuelmess.clock2.chronometer.ChronometerNotificationService.updateNotification(ChronometerNotificationService.java:356)
at com.emmanuelmess.clock2.timers.TimerNotificationService.handleDefaultAction(TimerNotificationService.java:201)
at com.emmanuelmess.clock2.chronometer.ChronometerNotificationService.onStartCommand(ChronometerNotificationService.java:218)
at com.emmanuelmess.clock2.timers.TimerNotificationService.onStartCommand(TimerNotificationService.java:171)
at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:3028)
at android.app.ActivityThread.-wrap17(ActivityThread.java)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1452)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:148)
at android.app.ActivityThread.main(ActivityThread.java:5443)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:728)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:618)
```

--- a/app/src/main/java/com/emmanuelmess/clock2/chronometer/ChronometerNotificationService.java
+++ b/app/src/main/java/com/emmanuelmess/clock2/chronometer/ChronometerNotificationService.java
@@ -1,4 +1,5 @@
 /*
+ * Copyright 2018 Emmanuel Messulam
  * Copyright 2017 Phillip Hsu
  *
  * This file is part of ClockPlus.
@@ -294,10 +295,7 @@ public abstract class ChronometerNotificationService extends Service {
      */
     @CallSuper
     protected void releaseResources(long id) {
-        mNoteBuilders.remove(id);
         quitThread(id);
-        mThreads.remove(id);
-        mDelegates.remove(id);
     }
 
     /**
@@ -394,5 +392,9 @@ public abstract class ChronometerNotificationService extends Service {
         if (thread != null && thread.isAlive()) {
             thread.quit();
         }
+
+        mNoteBuilders.remove(id);
+        mThreads.remove(id);
+        mDelegates.remove(id);
     }
 }

